community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java

```diff
@@ -289,7 +290,7 @@ public static void main( String[] incomingArguments, boolean defaultSettingsSuit
         }
         finally
         {
-            File badRelationships = new File( config.badFileName() );
+            File badRelationships = new File( storeDir, config.badFileName() );
```

Comment:
I don't think this works with absolute `--bad` filename paths as documented on the flag above.